glossary.yml

```diff
@@ -6362,6 +6427,14 @@
       Una secuencia de valores, normalmente de tipo [homogéneo](#homogeneous). Los
       vectores son la estructura de datos fundamental en [R](#r_language); un [escalar](#scalar) es
       solo un vector con exactamente un elemento.
+  ja:
+    term: "ベクトル"
+    def: >
+      順番を付けられたもののリストです。
```

Comment:
```suggestion
      順番を付けられた値のリスト（配列）のこと。
```

Reply:
Thanks! Is this what you were referring to here? Sorry I remembered our discussion on grammatical style but must’ve let some slip through. Otherwise small changes here and I think I understand it so hopefully means my Japanese is improving. 😊

> I followed our agreement on how to end the translation in Glosario (ref. #229 ) and suggested changes. Hope they make sense to you.